CRM/Contribute/Page/ContributionRecur.php

```diff
@@ -47,7 +47,7 @@ public function view() {
       ]);
     }
     catch (Exception $e) {
-      CRM_Core_Error::statusBounce('Recurring contribution not found (ID: ' . $this->getEntityId());
+      CRM_Core_Error::statusBounce(ts('Recurring contribution not found (ID: %1', [1 => $this->getEntityId()]));
```

Comment:
```suggestion
      CRM_Core_Error::statusBounce(ts('Recurring contribution not found (ID: %1)', [1 => $this->getEntityId()]));
```

The `)` was missing before but might as well fix it.